bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -28,215 +28,203 @@ spec:
         openshift.io/component: network
         kubernetes.io/os: "linux"
     spec:
-      # Requires fairly broad permissions - ability to read all services and network functions as well
-      # as all pods.
       serviceAccountName: ovn-kubernetes-controller
       hostNetwork: true
-      hostPID: true
       priorityClassName: "system-cluster-critical"
+      # volumes in all containers:
+      # (container) -> (host)
+      # /etc/openvswitch -> /var/lib/ovn - ovsdb schema (selinux blocks /etc on host)
+      # /var/lib/openvswitch -> /var/lib/ovn - ovsdb data
```

Comment:
As for `/etc/openvswitch`, it's just used for system-id.conf.. for now. You're right, that's sketchy and I'll change it.

As for the other renaming (/run/openvswitch -> /run/ovn), I did this because I wanted to be 100% certain there was no interaction between the master and node processes.